instrumentation/apache-httpclient/apache-httpclient-4.3/library/src/main/java/io/opentelemetry/instrumentation/apachehttpclient/v4_3/ApacheHttpClientTelemetryBuilder.java

```diff
@@ -35,6 +37,7 @@ public final class ApacheHttpClientTelemetryBuilder {
       httpAttributesExtractorBuilder =
           HttpClientAttributesExtractor.builder(ApacheHttpClientHttpAttributesGetter.INSTANCE);
   private boolean emitExperimentalHttpClientMetrics = false;
+  private Set<String> knownMethods = HttpConstants.KNOWN_METHODS;
```

Comment:
You could store the `HttpSpanNameExtractorBuilder` instead, same as the extractor (I guess this applies to every library instrumentation)